src/memtable.rs

```diff
@@ -194,18 +200,26 @@ impl MemTables {
     }
 
     /// Get mutable memtable
-    pub fn table_mut(&self) -> &MemTable {
-        &self.mutable
+    pub fn table_mut(&self) -> Arc<MemTable> {
```

Comment:
`xxx_mut` usually means a mutable refernece of xxx, better rename to `mut_table`